Source/WebKit/NetworkProcess/NetworkProcess.cpp

```diff
@@ -536,10 +540,13 @@ void NetworkProcess::addWebsiteDataStore(WebsiteDataStoreParameters&& parameters
         SandboxExtension::consumePermanently(*handle);
     if (auto& handle = parameters.parentBundleDirectoryExtensionHandle)
         SandboxExtension::consumePermanently(*handle);
-    if (auto& handle = parameters.tempDirectoryExtensionHandle)
+    if (auto& handle = parameters.tempDirectoryExtensionHandle) {
         SandboxExtension::consumePermanently(*handle);
+#if ENABLE(LLVM_PROFILE_GENERATION)
+        WTF::sendInitializeProfileCollectionNotification();
+#endif
```

Comment:
Ditto.

Reply:
No longer doing this in new implementation.